sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/ShareFileRequestIntent.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.storage.file.share.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/** Defines values for ShareFileRequestIntent. */
+public enum ShareFileRequestIntent {
```

Comment:
Why are there 2 versions of this file?  This may be confusion on how the Java generation process works on my part.